databases/sqlitebrowser/Portfile

```diff
@@ -45,59 +42,64 @@ github.livecheck.regex  {([0-9.]+)}
+                        QMAKE_LIBDIR+=${qt_libs_dir}
+
+    # Uses QSysInfo::buildAbi and QSysInfo::currentCpuArchitecture
+    qt5.min_version     5.4
+
+    pre-configure {
+        reinplace "s|libs/antlr-2.7.7/antlr.pro||g" ${worksrcpath}/sqlitebrowser.pro
+        reinplace "s|libs/qscintilla/Qt4Qt5/qscintilla.pro||g" ${worksrcpath}/sqlitebrowser.pro
+        reinplace "s|-lqscintilla2|-lqscintilla2_qt5|g" ${worksrcpath}/src/src.pro
+        reinplace "s|\$\$PWD/../libs/antlr-2.7.7||g" ${worksrcpath}/src/src.pro
+        reinplace "s|\$\$PWD/../libs/qscintilla/Qt4Qt5||g" ${worksrcpath}/src/src.pro
+        delete ${worksrcpath}/libs/antlr-2.7.7 ${worksrcpath}/libs/qscintilla
+    }
+
+    depends_lib-append  port:sqlite3
+    qt5.depends_component \
+                        qtmacextras qttools
+
+    destroot {
+            copy "${worksrcpath}/src/DB Browser for SQLite.app" ${destroot}${applications_dir}
+    }
 
     post-destroot {
-        set APPNAME "${qt_apps_dir}/SQLiteBrowser.app"
-        xinstall -m 755 -d ${destroot}${qt_apps_dir}
-        # for some reason the app PG capitalises the 1st letter
-        file rename ${destroot}${applications_dir}/Sqlitebrowser.app ${destroot}${APPNAME}
-        file rename ${destroot}${APPNAME}/Contents/MacOS/Sqlitebrowser ${destroot}${APPNAME}/Contents/MacOS/Sqlitebrowser-temp
-        file rename ${destroot}${APPNAME}/Contents/MacOS/Sqlitebrowser-temp ${destroot}${APPNAME}/Contents/MacOS/sqlitebrowser
-        file rename ${destroot}${APPNAME}/Contents/Resources/Sqlitebrowser.icns ${destroot}${APPNAME}/Contents/Resources/Sqlitebrowser.icns-temp
-        file rename ${destroot}${APPNAME}/Contents/Resources/Sqlitebrowser.icns-temp ${destroot}${APPNAME}/Contents/Resources/sqlitebrowser.icns
-        xinstall -m 755 ${worksrcpath}/src/app.plist ${destroot}${APPNAME}/Contents/Info.plist
-        reinplace "s|@EXECUTABLE@|sqlitebrowser|g" ${destroot}${APPNAME}/Contents/Info.plist
-        reinplace "s|@ICON@|sqlitebrowser.icns|g" ${destroot}${APPNAME}/Contents/Info.plist
-
-        if {[info procs qt5.add_app_wrapper] != ""} {
-            file delete ${destroot}${prefix}/bin/sqlitebrowser
-            qt5.add_app_wrapper sqlitebrowser SQLiteBrowser sqlitebrowser
-        }
+            ln -s "${applications_dir}/DB Browser for SQLite.app/Contents/MacOS/DB Browser for SQLite" ${destroot}${prefix}/bin/sqlitebrowser
     }
 }
+
 platform linux {
+    PortGroup       qt5 1.0
+    PortGroup       cmake 1.1
+
+    qt5.min_version 5.4
+
+    qt5.depends_component \
+                    qtmacextras qttools
     cmake.install_rpath-prepend \
                     ${qt_libs_dir} ${prefix}/lib
     configure.args-append \
-                    -DCMAKE_PREFIX_PATH=${prefix}
-    configure.args-replace \
-                    -DQT_LIBRARY_DIR=${qt_frameworks_dir} \
+                    -DCMAKE_PREFIX_PATH=${prefix} \
+                    -DQSCINTILLA_INCLUDE_DIR=${qt_includes_dir} \
```

Comment:
Maybe align this properly; somehow I missed that in my original version?

Reply:
Uh I don't see what's misaligned here? Or you mean they should be indented 25 spaces like other lines?